src/DependencyInjection/NeustaConverterExtension.php

```diff
@@ -62,7 +62,7 @@ private function registerConverterConfiguration(string $id, array $config, Conta
                     '$defaultValue' => $sourceConfig['default'] ?? null,
                     '$mapper' => null,
                     '$accessor' => new Reference('property_accessor'),
-                    '$skipNull' => $sourceConfig['skip_null'] || $skipNull,
+                    '$skipNull' => $skipNull || $sourceConfig['skip_null'],
```

Comment:
What's the reason for this change?

Reply:
I've changed it and afterwards tests were green. So I kept it without special regard.
After your comment I've tried to redo and tests are still green?!
So, I've changed it back.